synapse/state/__init__.py

```diff
@@ -424,7 +424,6 @@ async def compute_event_context_for_batched(
         self,
```

Comment:
this function previously added the state groups and information associated with the state groups to the event context, 
now it does not, meaning that callers will need to do that on their own. Is it valid to alert them to this in the docstring
and leave it at that?